cltk/text_reuse/automata.py

```diff
@@ -350,7 +354,7 @@ def transition_function(self, qi, u):
         except KeyError:
             return None
 
-    def to_DFA(self):
+    def ConvertToDeterministic(self):
```

Comment:
Please change to conform with PEP8
convert_to_deterministic
see: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names